utilities/transactions/optimistic_transaction_db_impl.h

```diff
@@ -29,9 +46,20 @@ class OptimisticTransactionDBImpl : public OptimisticTransactionDB {
                                 const OptimisticTransactionOptions& txn_options,
                                 Transaction* old_txn) override;
 
+  size_t GetLockBucketsSize() const { return bucketed_locks_.size(); }
+
+  OccValidationPolicy GetValidatePolicy() const { return validate_policy_; }
+
+  std::unique_lock<std::mutex> LockBucket(size_t idx);
+
  private:
+  // NOTE(wolfkdy): used in validation phase. Each key is hashed into some
```

Comment:
Do we need `NOTE(wolfkdy)` here?

Reply:
`wolfkdy` has been removed to match rocksdb's standard.